crypto/engine/eng_cryptodev.c

```diff
@@ -184,7 +186,6 @@ static struct {
     {
         CRYPTO_AES_CBC, NID_aes_256_cbc, 16, 32,
     },
-# ifdef CRYPTO_AES_CTR
```

Comment:
Are you really sure about this?  There must have been a reason why this guard was in place...